src/hugr/hugrmut.rs

```diff
@@ -60,6 +60,14 @@ pub trait HugrMut: HugrView + HugrMutInternals {
         self.hugr_mut().add_op_before(sibling, op)
     }
 
+    /// A generalisation of [`HugrMut::add_op_before`], needed temporarily until
+    /// add_op type methods all default to creating nodes with open extensions.
```

Comment:
Is there a github issue for this we could reference?

Reply:
#424 - I'll add it to the comment